sycl/CMakeLists.txt

```diff
@@ -599,3 +599,18 @@ add_custom_target(deploy-sycl-toolchain
 add_subdirectory(doc)
 # SYCL End-to-End tests
 add_subdirectory(test-e2e)
+
+add_custom_target(install-test-utilities ALL
```

Comment:
can we add sycl to the target name as this is not common to all the llvm tests? eg: llvm-size/llvm-profdata are not common.